### PR TITLE
Updating Accessibility Tree section

### PR DESCRIPTION
**Pull request by Steve-Noble**
Revising the accessibility tree text

### CODE REVIEW

**Review by Steve-Noble — COMMENTED** [review]
revising text

**Review by davidcarlisle — APPROVED** [review]
Looks fine, At some point, since this is saying what browsers should do, we'll probably need to distinguish MathML-Core from the rest. Since Non-Core elements are essentially [Unkown](https://w3c.github.io/mathml-core/#dfn-unknown-mathml-element) that may(?) limit how much we can mandate about their accessibility tree behaviour.  But happy for this to be merged at this stage

File: src/accessibility.html [hunk 1]
```diff
@@ -71,20 +71,15 @@ <h3 id="acc_guide">Accessibility Guidance</h3>
...
+ 
+  <p>By ensuring that MathML elements become nodes in the DOM tree, and the resulting accessibility tree, user agents can expose math nodes for keyboard navigation within expressions. This can support important user needs such as the ability to visually highlight elements of an expression and/or speak individual elements as one navigates with arrow keys. This can further support other forms of synchronous navigation, such as individuals using refreshable braille displays along with synthetic speech.</p> 
+
+  <p>While it is common practice for the accessibility tree to ignore most DOM node elements that are primarily used for visual display purposes, it is important to point out that math expressions often use what appears as visual styling to convey information which can be important for some types of assistive technology applications. For example, omitting the <mspace> element from the accessibility tree will impact the ability to generate a valid math braille representation of expressions on a braille display.</p>
```
Comment (dginev) [inline src/accessibility.html:81]: Should the `<mspace>` here be escaped to `&lt;mspace&gt;` ?

Comment (davidcarlisle) [reply src/accessibility.html:81]: yes or better `<code class="element">&lt;mspace/&gt;</code>`

**Pull request merged**